runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineOptions.java

```diff
@@ -23,16 +23,13 @@
 import org.apache.beam.sdk.options.DefaultValueFactory;
 import org.apache.beam.sdk.options.Description;
 import org.apache.beam.sdk.options.PipelineOptions;
-import org.apache.beam.sdk.options.StreamingOptions;
-
-
 
 /**
  * Spark runner {@link PipelineOptions} handles Spark execution-related configurations,
  * such as the master address, batch-interval, and other user-related knobs.
  */
 public interface SparkPipelineOptions
-    extends PipelineOptions, StreamingOptions, ApplicationNameOptions {
```

Comment:
I wouldn't remove the `isStreaming` option, but set it instead.
For one, SDK tests for streaming would require "forcing" streaming on bounded sources, and that it communicated via `PipelineOptions.isStreaming()`